api/catalog/api/serializers/media_serializers.py

```diff
@@ -117,13 +117,14 @@ def validate_q(self, value):
     def validate_license(value):
         """Checks whether license is a valid license code."""
 
-        licenses = value.lower().split(",")
+        value = value.lower()
+        licenses = value.split(",")
```

Comment:
Nit: I would just combine these lines
```suggestion
        licenses = value.lower().split(",")
```